### PR TITLE
Content warnings

### LINKED ISSUE #68: Show Warnings
Should be able to add warnings to shows like smoking, strobe, foul dirty language, etc to appear on the website and screens to warn people a little more in advance than after they have bought their tickets,

### PR DESCRIPTION
**Pull request by samozzy**
- Shows get content warnings
- These are visible on the FOH show report page, and the booking page.
- The booking page can have a subset of the FOH page (selective by warning, not by show)
- Warnings are categorised by action, dialogue and technical; ordered by category, then alphabetically.

To close #68

### CODE REVIEW

**Comment by samozzy** [comment]
I think, after a year, this might actually be ready to merge...!

**Comment by samozzy** [comment]
The fact that warnings are selective by warning and not by show seems a bit daft in hindsight.
Option 1: Granular warning visibility by show, if this is even possible
Option 2: Show toggle for displaying warnings to audience
Option 3: All visible, anything else in the 'warning notes' field visible at box office 

--> Decided on option 3 for a combination of ease and discouraging the hiding of content warnings unnecessarily

File: tickets/migrations/0005_auto_20180608_1554.py [hunk 1]
```diff
@@ -8,7 +8,7 @@
 class Migration(migrations.Migration):
 
     dependencies = [
-        ('tickets', '0004_auto_20161024_1658'),
+        ('tickets', '0005_auto_20171031_2322'),
```
Comment (davidtaylorhq) [inline tickets/migrations/0005_auto_20180608_1554.py:11]: This looks suspicious... why is an old migration being modified?

Comment (samozzy) [reply tickets/migrations/0005_auto_20180608_1554.py:11]: Master had a migration after this branch was started locally which was causing travis to fail

Comment (davidtaylorhq) [reply tickets/migrations/0005_auto_20180608_1554.py:11]: Problem is, 0005_auto_20171031_2322 doesn't exist anywhere... as far as I can see...

Comment (samozzy) [reply tickets/migrations/0005_auto_20180608_1554.py:11]: [Is part of this PR](https://github.com/newtheatre/nt-tickets/pull/115/files#diff-ec28dd054a39e11c7c478fd633658fdf)

**Review by davidtaylorhq — APPROVED** [review]

**Pull request merged**